### PR TITLE
Update Ingest sheet page details info and general cleanup

### PR DESCRIPTION
**Pull request by adamjarling**
@vlsrobinson we can always update this more in the future, but not sure how valuable some of the info would be?

image

### CODE REVIEW

**Comment by kdid** [comment]
I ingested 500 works but it says Total Works 10, which is the number it's previewing.

**Comment by adamjarling** [comment]
Ah, one sec.

**Comment by vlsrobinson** [comment]
Looks great! I think the 'complete', preview of works and total works created is  good for now.  We can see if the Prod team wants additional info at some point in the future after we get it in front of them (like # of file sets or something.)  But I would just want it as a quick double-check.

**Comment by adamjarling** [comment]
@kdid So the 10 value is coming back because we're limiting the results as part of the GraphQL query.  https://github.com/nulib/meadow/blob/9a2496a19dab762667c8af88922957c6cca3b281/assets/js/components/IngestSheet/Completed.jsx#L23

And we put this in place because returning all the rows was breaking the UI in big number scenarios, right?  So we need some way to get an aggregate count of this from GraphQL, without returning the data.   Looks like GraphQL doesn't support this out of the box, but maybe we create a new Query which just lists the total count?  

I'm not seeing any obvious place we can get this data at the moment?   Maybe we just suppress the "Details" section for now?

**Comment by adamjarling** [comment]
Or maybe we grab it from ElasticSearch for successful ingests?

**Comment by kdid** [comment]
Yes, can get that from Elasticsearch

File: assets/js/components/IngestSheet/Completed.jsx [hunk 1]
```diff
@@ -29,6 +31,37 @@ const IngestSheetCompleted = ({ sheetId, title }) => {
...
+            ],
+          },
+        },
+        size: 0,
+      });
+      setTotalWorks(response.hits.total);
+    };
+    //fn();
+  }, [sheetId]);
+
```
Comment (adamjarling) [inline assets/js/components/IngestSheet/Completed.jsx:64]: @kdid This is where we're now grabbing the info direct from Elasticsearch.

Comment (kdid) [reply assets/js/components/IngestSheet/Completed.jsx:64]: Is it working for you locally? I'm seeing your getting results from Elasticsearch but I'm not seeing it show in the UI.

Comment (kdid) [reply assets/js/components/IngestSheet/Completed.jsx:64]: also `yarn.lock` conflict

Comment (adamjarling) [reply assets/js/components/IngestSheet/Completed.jsx:64]: @kdid Just fixed up both now.  I think I messed up a local stash before the last push, or something weird.  Everything should display now and blew away the `yarn.lock` for a fresh copy.

**Review by kdid — APPROVED** [review]

**Pull request merged**